### ISSUE TITLE
Recognition for reviewers

### ISSUE THREAD

**Issue by annethyme** [act-rules/act-rules.github.io#239]
While working on documentation for Auto-WCAG, I realised that the reviewers are totally invisible as soon as a rule is published. 

The authors get recognition for the rule, but the reviewers are not mentioned, which means that they:

1. will not be shown any appreciation for their effort, and 
2. will not either be held responsible if they let something go through that should have been held back. 

## Suggestion
I suggest that we consider listing the reviewers on a rule, as well as the authors - only the reviewers making the final 3 or more approvals, though. 

## Anticipated outcome
This would create an incentive for reviewers to follow the rule all the way through the iterations process, and ensure that people don't approve anything they don't want associated with their name.

Hopefully this could help us increase number of reviewers and the activity level for those doing reviews, as well as the thoroughness of the reviews.

**Comment by nitedog** [comment]
I agree with this. However, this may start getting lengthy. Maybe we should move this section to the bottom of a rule as "Acknowledgements" with different subsections:

- Authors
- Reviewers
- (if applicable) Funding Sources or Donations

**Comment by WilcoFiers** [comment]
Moving this off the September agenda. We need to prioritise open PRs. We'll pick it up in October.

**Comment by annethyme** [comment]
Additional suggestion:
Create a "high score" for rule authors and one for reviewers to make it visible, who spends all of their time working within this project.
Remember to list which organisations people belong to, to create an incentive for an organisation to contribute more. 

Also consider if we need to be able to e.g. pull list by year.

**Comment by WilcoFiers** [comment]
I'll create a PR using Shadi's proposal.

**Comment by jkodu** [comment]
@WilcoFiers - was a PR created for this? Just following up.

**Comment by annethyme** [comment]
There is now an "Akcnowledgements" section (optional) in the latest editor's draft for the ACT Rules Format, https://w3c.github.io/wcag-act/act-rules-format.html#act-rule-structure.

@jkodu closed issue #350 about aligning sections between ACT Rules Format and Auto-WCAG, since only part left on that one was this.

**Comment by jkodu** [comment]
@WilcoFiers - are we looking to automagically find the merge commit for each rule, and find the relevant PR to find all the reviewers?

**Comment by WilcoFiers** [comment]
This is a technically complicated thing to develop. I'm not in favour of putting resources towards developing this. If anyone has the time to do so they are welcome to, but short of that, :-1: from me.

**Comment by annethyme** [comment]
Suggestion for an easy way to do this: 
Whenever a rule has passed Final Call, the author edits the rule to add everyone who has currently approved the rule to a "Reviewers" list right under the "Authors" list.

**Comment by WilcoFiers** [comment]
Discussed on the call to add a "previous authors" field to the frontmatter of rules.

**Comment by jkodu** [comment]
Conclusion is to change the frontmatter to something like below:

```md
acknowledgements:
	authors:
		- author1
	previous_authors:
		- someone
```
